newdle/client/src/components/MyNewdles.js

```diff
@@ -1,18 +1,55 @@
-import React from 'react';
-import {Container, Icon, Label} from 'semantic-ui-react';
+import React, {useEffect, useState} from 'react';
+import {Container, Icon, Label, Placeholder} from 'semantic-ui-react';
 import {useHistory} from 'react-router';
 import {serializeDate, toMoment} from '../util/date';
 import styles from './MyNewdles.module.scss';
+import client from '../client';
 
-function MyNewdle({
-  newdle: {code, title, participantsCount, participantsTotal, duration, finalDt, timezone},
-}) {
+export default function MyNewdles() {
+  const [newdles, setNewdles] = useState(null);
+
+  useEffect(() => {
```

Comment:
I did not put this in redux since the data is only needed on this page, so using redux here would just make things more complex.